results/Snowflake__snowflake-arctic-embed-l-v2.0/edc2df7b6c25794b340229ca082e7c78782e6374/CMedQAv2-reranking.json

```diff
@@ -0,0 +1,26 @@
+{
+  "dataset_revision": "23d186750531a14a0357ca22cd92d712fd512ea0",
+  "task_name": "CMedQAv2-reranking",
+  "mteb_version": "1.23.0",
+  "scores": {
+    "test": [
+      {
+        "map": 0.70069,
+        "mrr": 0.748533,
+        "nAUC_map_max": 0.598828,
+        "nAUC_map_std": 0.31886,
+        "nAUC_map_diff1": 0.541323,
+        "nAUC_mrr_max": 0.694851,
+        "nAUC_mrr_std": 0.382436,
+        "nAUC_mrr_diff1": 0.634483,
+        "main_score": 0.70069,
+        "hf_subset": "default",
+        "languages": [
+          "cmn-Hans"
+        ]
+      }
+    ]
+  },
+  "evaluation_time": 33.124486207962036,
+  "kg_co2_emissions": null
```

Comment:
I am a bit sad not to have the co2 usage, but otherwise, this looks fine

Reply:
Is there any reason why `codecarbon` is currently not enabled by default? I'll modify my script to enable it from now on on new runs.

Reply:
Actually, I'll just run it again with it enabled, it doesn't take long.